[ADD] Marketing Card: Add new doc (#17662)
odoo/documentation: content/applications/marketing/marketing_card.rst

marketers can design dynamic card templates using different layouts. They can also expand their
reach by emailing cards to recipients in bulk and allowing recipients to share them with their own
networks.

.. _marketing_card/card-campaigns:

Card campaigns
==============

To see a dashboard of all created card campaigns, navigate to the **Marketing Card** app. By
default, card campaigns appear in the :icon:`oi-view-list` :guilabel:`(List)` view, providing users
with detailed information about each campaign. Alternatively, the :icon:`oi-view-kanban`
:guilabel:`(Kanban)` view provides users with a card view of all campaigns, along with icons
displaying each campaign's :ref:`performance metrics <marketing_card/metrics>`.

.. _marketing_card/create-campaign:

Create a campaign
=================

To create a campaign, click :guilabel:`New`. This opens a new campaign form, allowing users to
configure campaign information, design the :ref:`card layout
<marketing_card/create-campaign/card-layout>`, and customize the :ref:`campaign message
<marketing_card/create-campaign/recipient-message>`.

.. image:: marketing_card/marketing-card-form.png

>    :alt: New marketing card campaign form in Odoo Marketing Card.

Reviewer: i like that a fully configured example of a marketing card is displayed after the description of what it can do! (:

Begin by providing a name for the campaign in the :guilabel:`Name` field. This field is
**required**.

Then, in the :guilabel:`Recipients` drop-down menu, users **must** select the type of recipient of
the campaign. The following options are provided:

- :guilabel:`Contact`: Send cards to existing contacts.
- :guilabel:`Event Booth`: Send cards to contacts with registered event booths.
- :guilabel:`Event Registration`: Send cards to contacts with event registrations.

> - :guilabel:`Event Track`: Send cards to contacts with event tracks.

Reviewer: This might be because I'm unfamiliar with what this app does, but as a new user I don't think I'm given enough context to confidently fill out Recipients.

Right now, the options describe *who* receives the cards, but not *why* a user might choose one recipient type over another or what these records represent in practice. For example, if someone isn’t already familiar with Events app terminology, they might not know what "event booth" or "event track" mean off the bat. 

I don’t think this needs a lot more detail, but a little more context would help orient the reader. A good starting point could be:
- adding inline links to the Event Booth, Event Registration, and Event Track documentation when mentioned
- briefly explaining the use case for each recipient type (e.g., why someone would send cards to booth contacts vs. event registrants)

I’ll leave it to you to find the right balance between concise and explanatory!

Once selected, a :guilabel:`Preview on...` field appears, allowing the user to preview the campaign

> on a specific contact from the selected recipient type. After choosing a preview target, a

Reviewer: I may have gotten too in the weeds of the implementation details (and dragged Ethan into it too 💀 ), but maybe there is something we can clarify here!

To me, the current doc as written reads to me that the `Recipients` field defines the actual people receiving the campaign. And, it might be the way I read docs while setting stuff up in the database, but I read up to line 44 and was trying to fill out the Recipient field. I had yet to get to your explanation about what the "Preview on..." field does (wah 😢 ) on line 46 and misunderstood while filling out the field in runbot that "Preview on" was there to further filter the recipient list.

It might just be me, but do you think it would be helpful to give a quick overview of the workflow  around line 30, or line 38? Personally I was lost until you clarified in our group chat with Ethan:

> the "recipients" field targets the type (e.g., contacts, registrations, booths) and then the email form is where you would specify/filter the actual contacts

So maybe slightly modifying the language here to more directly explain that Recipient is really defining the type/source of records the campaign targets, and it can target all event registrations, booth-related records, track-related records. And that the final recipient list is actually configured later in the email form/filter section (maybe with an internal link forward to that part of the doc?).

:guilabel:`Preview` button appears above the campaign form.

Next, fill out the following fields:

- :guilabel:`Post Link`: Relative URL of a page on the user's Odoo website.
- :guilabel:`Post Suggestion`: Description below the card when sharing on X.

> - :guilabel:`Responsible`: User responsible for the card campaign.

Reviewer: it might be helpful to briefly mention whether assigning a responsible user has any functional impact

For example, in the eLearning app, the responsible user receives notifications for new comments added below the tutorial. If there's similar notification or follow-ups that get sent to that user for marketing cards, let's include that context (:

---------------

The :guilabel:`Card Layout` tab on the campaign form provides options to configure the card's text
and appearance.

To customize the card, users can configure the following fields:

- :guilabel:`Background`: Upload an image to be displayed as the card background.
- :guilabel:`Header`: Enter the main header text.
- :guilabel:`Sub-Header`: Enter a sub-header text, displayed above the header.
- :guilabel:`Section`: Enter a text to be displayed in the lower section.
- :guilabel:`Sub-section 1`: Enter a text to be displayed below the lower section.
- :guilabel:`Sub-section 2`: Enter a text to be displayed next to :guilabel:`Sub-section 1`.
- :guilabel:`Dynamic Image 1`: Select an existing image from the selected recipient's model to be
  displayed in a corner of the card. This field only takes a :ref:`dynamic placeholder
  <marketing_card/create-campaign/card-layout/dynamic-placeholders>` as input.
- :guilabel:`Dynamic Image 2`: Select an existing image from the selected recipient's model to be
  displayed in a corner of the card. This field only takes a :ref:`dynamic placeholder
  <marketing_card/create-campaign/card-layout/dynamic-placeholders>` as input.

> - :guilabel:`Button`: Enter a text for the button.

Reviewer: ```suggestion
- :guilabel:`Button`: Enter the button label text.
```

or alternatively, `Enter the text displayed on the button.` for a non-technical audience

A preview of the resulting card allows the user to interactively visualize their configuration.
Above the preview are selectable themes to change the appearance of the card.

> 

Reviewer: the image mockup from the R&D spec cannot be published, but it might be helpful at a later date to know where each of these fields show up on the card in relation to each other

- :guilabel:`Reward Link`: Provide a link to reward recipients who share the campaign.
- :guilabel:`Thank You Message`: Enter a message to thank recipients for sharing the campaign.

.. image:: marketing_card/recipient-message-tab.png
   :alt: Recipient Message tab in the card campaign form in Odoo Marketing Card.

.. _marketing_card/preview:

Preview campaign
================

Once the card campaign form has been configured, click the :guilabel:`Preview` button to view a copy
of the campaign to be sent to recipients.

The campaign layout is organized with the request, and its description displayed at the very top,
followed by the card, then a list of social media icons that allow the user to share the campaign
across different platforms.

.. image:: marketing_card/card-campaign-preview.png
   :alt: Preview of a card campaign in Odoo Marketing Card.

.. _marketing_card/mailing:

Campaign mailing
================

After configuring a card campaign, users can :ref:`configure mailing options
<marketing_card/mailing/configure>`, then :ref:`share the cards <marketing_card/mailing/send>` with
recipients.

.. _marketing_card/mailing/configure:

Configure mailing
-----------------

> To configure the campaign mailing, click :guilabel:`Send`. On the :guilabel:`Send Cards` pop-up

Reviewer: ```suggestion
To configure the campaign mailing, on the marketing card form, click :guilabel:`Send`. On the :guilabel:`Send Cards` pop-up
```

feel free to change the wording! I just thought to improve the sections for skimmability, we start this section by reminding which page we're on before telling them what button to press

window, users configure the :ref:`email form <email_marketing/create_email>` to filter or specify
recipients and customize the body of the message.

.. image:: marketing_card/send-cards-form.png
   :alt: Send Cards window when sending mailing campaigns in Odoo Marketing Card.

To start, enter the subject of the mailing in the :guilabel:`Subject` field.

The :guilabel:`Card Campaign` and :guilabel:`Recipients` fields are already populated. The filtering
rule under the :guilabel:`Recipients` field can be modified to update the mailing list.

The :guilabel:`Mail Body` tab is automatically populated with a default message when the card is
created. This message can be modified using the website editor.

After configuring the email form, click the :guilabel:`Update (#) Cards` button to generate or
update the personalized cards for each recipient. Then, on the :guilabel:`Confirm Cards Update`
pop-up window, click :guilabel:`Update Cards`, or click :guilabel:`Cancel` to cancel.

> 

Reviewer: this isn't the "happy path" that you're currently documenting, but what does clicking "Save" or "Discard" at the bottom of the "Send Cards" popup do?

<img width="979" height="988" alt="Image" src="https://github.com/user-attachments/assets/9001ea9b-056e-4c08-9272-c181e3065771" />

might be good to include a quick explanation in a note block? (:

.. _marketing_card/mailing/send:

Send cards
----------

Once the cards have been updated, users can either send the cards or schedule the mailing for a
future date.

> 

Reviewer: what you're currently documenting is the end-to-end workflow of creating and sending a marketing card for the first time, and I think the doc does that well! 😊 

One small improvement that could make the doc even more useful for returning users (and improve skimmability) would be including a quick navigation note explaining how to access existing mailings/cards to get to this mailing page you're on currently. 

From my testing, it seems to be done by going to Marketing Cards > Campaigns, selecting the card, clicking the Mailings smart button, and opening the specific record. (feel free to replace this path if you find one that has fewer steps 😅 )

==============
Marketing Card
==============

The **Marketing Card** application allows users to create and manage promotional campaigns with

> personalized, digital cards as well as view analytics about their campaign performance. Event

Reviewer 2: ```suggestion
personalized, digital cards, as well as view analytics about their campaign performance. Event
```
suggestion

marketers can design dynamic card templates using different layouts. They can also expand their

> reach by emailing cards to recipients in bulk and allow recipients to share the cards to their own

Reviewer 2: ```suggestion
reach by emailing cards in bulk to recipients and allow recipients to share the cards to their own
```
suggestion

> marketers can design dynamic card templates using different layouts. They can also expand their
> reach by emailing cards to recipients in bulk and allow recipients to share the cards to their own
> networks.

Reviewer 2: ```suggestion
marketers can design dynamic card templates using different layouts. They can also expand their
reach by emailing cards to recipients in bulk and allowing recipients to share them with their own
networks.
```
suggestion/ correction - changed 'to' with 'with'

.. _marketing_card/card-campaigns:

Card campaigns
==============

To see a dashboard of all created card campaigns, navigate to the **Marketing Card** app. By
default, card campaigns appear in the :icon:`oi-view-list` :guilabel:`(List)` view, providing users
with detailed information about each campaign. Alternatively, the :icon:`oi-view-kanban`

> :guilabel:`(Kanban)` view provides users with a card-view of all campaigns, along with icons

Reviewer 2: ```suggestion
:guilabel:`(Kanban)` view provides users with a card view of all campaigns, along with icons
```
suggestion. I am sure your section already uses that spelling. If it's already established in the docs, ignore this.

displaying each campaign's :ref:`performance metrics <marketing_card/metrics>`.

.. _marketing_card/create-campaign:

Create a campaign
=================

To create a campaign, click :guilabel:`New`. This opens a new campaign form, allowing users to
configure campaign information, design the :ref:`card layout
<marketing_card/create-campaign/card-layout>`, and customize the :ref:`campaign message
<marketing_card/create-campaign/recipient-message>`.

.. image:: marketing_card/marketing-card-form.png
   :alt: New marketing card campaign form in Odoo Marketing Card.

Begin by providing a name for the campaign in the :guilabel:`Name` field. This field is
**required**.

Then, in the :guilabel:`Recipients` drop-down menu, users **must** select the type of recipient of
the campaign. The following options are provided:

- :guilabel:`Contact`: Send cards to existing contacts.

> - :guilabel:`Event Booth`: Send cards contacts with registered event booths.

Reviewer 2: ```suggestion
- :guilabel:`Event Booth`: Send cards to contacts with registered event booths.
```
Correction: Not sure if this is correct, but it seems to be missing a word.

- :guilabel:`Event Registration`: Send cards to contacts with event registrations.
- :guilabel:`Event Track`: Send cards to contacts with event tracks.

Once selected, a :guilabel:`Preview on...` field appears, allowing the user to preview the campaign
on a specific contact from the selected recipient type. After choosing a preview target, a
:guilabel:`Preview` button appears above the campaign form.

Next, fill out the following fields:

- :guilabel:`Post Link`: Relative URL of a page on the user's Odoo website.
- :guilabel:`Post Suggestion`: Description below the card when sharing on X.
- :guilabel:`Responsible`: User responsible for the card campaign.
- :guilabel:`Tags`: Relevant tags for the campaign.

.. _marketing_card/create-campaign/card-layout:

Card Layout tab
---------------

> The :guilabel:`Card Layout` tab on the campaign form provides options to configure the text and
> appearance of the card.

Reviewer 2: ```suggestion
The :guilabel:`Card Layout` tab on the campaign form provides options to configure the card's text and
appearance.
```
suggestion

- :guilabel:`Sub-Header`: Enter a sub-header text, displayed above the header.
- :guilabel:`Section`: Enter a text to be displayed in the lower section.
- :guilabel:`Sub-section 1`: Enter a text to be displayed below the lower section.
- :guilabel:`Sub-section 2`: Enter a text to be displayed next to :guilabel:`Sub-section 1`.
- :guilabel:`Dynamic Image 1`: Select an existing image from the selected recipient's model to be
  displayed in a corner of the card. This field only takes a :ref:`dynamic placeholder
  <marketing_card/create-campaign/card-layout/dynamic-placeholders>` as input.
- :guilabel:`Dynamic Image 2`: Select an existing image from the selected recipient's model to be
  displayed in a corner of the card. This field only takes a :ref:`dynamic placeholder
  <marketing_card/create-campaign/card-layout/dynamic-placeholders>` as input.
- :guilabel:`Button`: Enter a text for the button.

A preview of the resulting card allows the user to interactively visualize their configuration.
Above the preview are selectable themes to change the appearance of the card.

.. _marketing_card/create-campaign/card-layout/dynamic-placeholders:

Dynamic placeholders
~~~~~~~~~~~~~~~~~~~~

> By default, the text content of each field is static. However, the fields listed (except for

Reviewer 2: ```suggestion
By default, each field's text content is static. However, the fields listed (except for
```
suggestion

:guilabel:`Background` and :guilabel:`Button`) can also be populated automatically using
:ref:`dynamic placeholders <email_template/dynamic-placeholders>` to personalize the card content
for different recipients.

> To do this, click the checkmark next to the field. Then, browse and select a placeholder value to be

Reviewer 2: ```suggestion
To do this, click the check mark next to the field. Then, browse and select a placeholder value to be
```
correction unless the GUI spellss it as checkmark.

populated from the selected recipient.

.. _marketing_card/create-campaign/recipient-message:

Recipient Message tab
---------------------

The :guilabel:`Recipient Message` tab on the campaign form provides options to configure messages
sent to recipients when sharing the card.

Users can configure the following fields:

- :guilabel:`Request`: Enter a request to be displayed at the top of the campaign.
- :guilabel:`Request Description`: Enter a description of the request, displayed below the request.
- :guilabel:`Reward Link`: Provide a link to reward recipients who share the campaign.
- :guilabel:`Thank You Message`: Enter a message to thank recipients for sharing the campaign.

.. image:: marketing_card/recipient-message-tab.png
   :alt: Recipient Message tab in the card campaign form in Odoo Marketing Card.

.. _marketing_card/preview:

Preview campaign
================

Once the card campaign form has been configured, click the :guilabel:`Preview` button to view a copy

> of a campaign to be sent to recipients.

Reviewer 2: ```suggestion
of the campaign to be sent to recipients.
```
correction

> The campaign layout is organized with the request and the request description displayed at the very
> top, followed by the card, then a list of social media icons allowing the user to share the campaign
> across different platforms.

Reviewer 2: ```suggestion
The campaign layout is organized with the request, and its description displayed at the very
top, followed by the card, then a list of social media icons that allow the user to share the campaign
across different platforms.
```
suggestion

.. image:: marketing_card/card-campaign-preview.png
   :alt: Preview of a card campaign in Odoo Marketing Card.

.. _marketing_card/mailing:

Campaign mailing
================

After configuring a card campaign, users can :ref:`configure mailing options

> <marketing_card/mailing/configure>` then :ref:`share the cards <marketing_card/mailing/send>` with

Reviewer 2: ```suggestion
<marketing_card/mailing/configure>`, then :ref:`share the cards <marketing_card/mailing/send>` with
```
suggestion

recipients.

.. _marketing_card/mailing/configure:

Configure mailing
-----------------

To configure the campaign mailing, click :guilabel:`Send`. On the :guilabel:`Send Cards` pop-up
window, users configure the :ref:`email form <email_marketing/create_email>` to filter or specify
recipients and customize the body of the message.

.. image:: marketing_card/send-cards-form.png
   :alt: Send Cards window when sending mailing campaigns in Odoo Marketing Card.

To start, enter the subject of the mailing in the :guilabel:`Subject` field.

The :guilabel:`Card Campaign` and :guilabel:`Recipients` fields are already populated. The filtering
rule under the :guilabel:`Recipients` field can be modified to update the mailing list.

> The :guilabel:`Mail Body` tab is automatically populated with a default message along with the card.

Reviewer 2: ```suggestion
The :guilabel:`Mail Body` tab is automatically populated with a default message when the card is created.
```
suggestion

This message can be modified using the website editor.

After configuring the email form, click the :guilabel:`Update (#) Cards` button to generate or
update the personalized cards for each recipient. Then, on the :guilabel:`Confirm Cards Update`
pop-up window, click :guilabel:`Update Cards`, or click :guilabel:`Cancel` to cancel.

.. _marketing_card/mailing/send:

Send cards
----------

Once the cards have been updated, users can either send the cards or schedule the mailing for a
future date.

To send the mailing, click :guilabel:`Send`. On the :guilabel:`Ready to unleash emails?` pop-up
window, click :guilabel:`Send to all` to send the cards to the specified recipients.

Alternatively, to schedule for a later date, click :guilabel:`Schedule`. On the pop-up window,
select a date in the :guilabel:`Send on` field. Finally, click :guilabel:`Schedule` to schedule the
mailing.

> After sending or scheduling a mailing campaign, a series of smart buttons appear, showing the

Reviewer 2: ```suggestion
After sending or scheduling a mailing campaign, a series of smart buttons appears, showing the
```
correction

:doc:`engagement metrics <email_marketing/analyze_metrics>` for the campaign.

.. _marketing_card/metrics:

Campaign metrics
================

> At any point, users can view metrics about the campaign, displayed as smart buttons at the top of

Reviewer 2: ```suggestion
At any time, users can campaign metrics, displayed as smart buttons at the top of
```
suggestion

the campaign form.

The following buttons provide specific information about the performance of the campaign:

- :icon:`fa-envelope` :guilabel:`Mailings`: View a list of recipients of the campaign.
- :icon:`fa-mouse-pointer` :guilabel:`Clicks`: View how many times the campaign was clicked. This
  button is **not** clickable.
- :icon:`fa-paper-plane` :guilabel:`Cards`: View how many cards were sent to recipients.
- :icon:`fa-eye` :guilabel:`Opened`: View how many times the campaign was opened.
- :icon:`fa-share` :guilabel:`Shared`: View how many times the campaign was shared.

> Clicking on a smart button opens a list of occurrences or instances of the metric (e.g., a list of

Reviewer 2: ```suggestion
Clicking a smart button opens a list of occurrences or instances of the metric (e.g., a list of
```
correction
